### PR TITLE
Fixed card insertion after removing first card

### PR DESCRIPTION
**Pull request by fantoine**
This issue occures when removing the very first card with removeCardInIndexRange.
The `index` variable used to get a visible card is equals to 0. Then it tries to get index `-1` of the `visibleCards` array.

### CODE REVIEW

**Comment by fantoine** [comment]
Hi, do you think you could check this PR?
I think it may fix some issues on cards removing.

File: Pod/Classes/KolodaView/KolodaView.swift [hunk 1]
```diff
@@ -653,6 +657,11 @@ public class KolodaView: UIView, DraggableCardDelegate {
         let visibleIndexes = [Int](indexRange).filter { $0 >= currentCardIndex && $0 < currentCardIndex + countOfVisibleCards }
         if !visibleIndexes.isEmpty {
             proceedDeletionInRange(visibleIndexes[0]..<visibleIndexes[visibleIndexes.count - 1])
+            
+            // Update current card index if the first visible card has been removed
+            if visibleIndexes[0] == currentCardIndex {
```
Comment (AEugene) [inline Pod/Classes/KolodaView/KolodaView.swift]: This is fixed in #190

**Comment by AEugene** [comment]
Hi @fantoine Everything is great, except indexing in deletion of cards. It is fixed in another request.

**Comment by fantoine** [comment]
Hi @AEugene, that's even perfect like this. I hope you'll release a new patch soon.

**Comment by AEugene** [comment]
@fantoine Could you remove lines 660-662 and I'll merge your request after #190

**Comment by fantoine** [comment]
@AEugene Done

**Comment by fantoine** [comment]
@AEugene I saw you merged #190. Do you think you could also merge this PR?

File: Pod/Classes/KolodaView/KolodaView.swift [hunk 2]
```diff
@@ -657,7 +657,7 @@ public class KolodaView: UIView, DraggableCardDelegate {
         if !visibleIndexes.isEmpty {
             proceedDeletionInRange(visibleIndexes[0]...visibleIndexes[visibleIndexes.count - 1])
         }
-        currentCardIndex -= Array(indexRange).filter { $0 < currentCardIndex }.count
+        currentCardIndex += Array(indexRange).filter { $0 < currentCardIndex }.count
```
Comment (AEugene) [inline Pod/Classes/KolodaView/KolodaView.swift]: Let me explain. Cards can be swiped programatically only with `.swipe(.direction)` method. 
In this case you want to delete unused cards both from Koloda and your data source, so they are synced.
For example, here is your data source:` [orangeCard, yellowCard, redCard, blueCard, greenCard]`
`currentItemIndex == 2 ` - so redCard is on top.
you remove orangeCard, yellowCard from your data source and you call Koloda `remove` method at indexes 0...1 .
Now your data source state is :  ` [redCard, blueCard, greenCard]`
So redCard is still on top and Koloda has `currentItemIndex == 0 `

Comment (serejahh) [inline Pod/Classes/KolodaView/KolodaView.swift]: Do you mean the hunk should be discarded?

**Pull request closed without merging**